app/src/atoms/buttons/OnDeviceDisplay/LargeButton.tsx

```diff
@@ -0,0 +1,128 @@
+import * as React from 'react'
+import { css } from 'styled-components'
+import {
+  TYPOGRAPHY,
+  COLORS,
+  SPACING,
+  BORDERS,
+  NewPrimaryBtn,
+  styleProps,
+  DIRECTION_ROW,
+  Icon,
+} from '@opentrons/components'
+import { StyledText } from '../../text'
+import type { StyleProps } from '@opentrons/components'
+
+type LargeButtonTypes = 'primary' | 'secondary' | 'alert'
+interface LargeButtonProps extends StyleProps {
+  onClick: () => void
+  buttonType: LargeButtonTypes
+  buttonText: React.ReactNode
+  disabled?: boolean
+}
+
+export function LargeButton(props: LargeButtonProps): JSX.Element {
+  const { onClick, buttonType, buttonText, disabled } = props
+  const buttonProps = {
+    onClick,
+    disabled,
+  }
+
+  const LARGE_BUTTON_PROPS_BY_TYPE: Record<
+    LargeButtonTypes,
+    {
+      defaultBackgroundColor: string
+      activeBackgroundColor: string
+      defaultColor: string
+      iconColor: string
+    }
+  > = {
+    secondary: {
+      defaultColor: COLORS.darkBlackEnabled,
+      defaultBackgroundColor: COLORS.foundationalBlue,
+      //  TODO(jr, 3/20/23): replace these hex codes with the color constants
+      activeBackgroundColor: '#99b1d2',
+      iconColor: COLORS.blueEnabled,
+    },
+    alert: {
+      defaultColor: COLORS.red_one,
+      defaultBackgroundColor: COLORS.red_three,
+      activeBackgroundColor: '#c8acad',
+      iconColor: COLORS.red_one,
+    },
+    primary: {
+      defaultColor: COLORS.white,
+      defaultBackgroundColor: COLORS.blueEnabled,
+      activeBackgroundColor: '#2160ca',
+      iconColor: COLORS.white,
+    },
+  }
+
+  const LARGE_BUTTON_STYLE = css`
+    text-align: ${TYPOGRAPHY.textAlignLeft};
+    color: ${LARGE_BUTTON_PROPS_BY_TYPE[buttonType].defaultColor};
+    background-color: ${LARGE_BUTTON_PROPS_BY_TYPE[buttonType]
+      .defaultBackgroundColor};
+    cursor: default;
+    border-radius: ${BORDERS.size_four};
+    box-shadow: none;
+    padding: ${SPACING.spacing5} ${SPACING.spacing5} 2.4375rem
+      ${SPACING.spacing5};
```

Comment:
top | left & right | bottom

```suggestion
    padding: ${SPACING.spacing5} ${SPACING.spacing5} 2.4375rem;
```